packages/web-app-markdown-editor/src/App.vue

```diff
@@ -98,14 +143,44 @@ export default {
       return loadFileTask.isRunning || saveFileTask.isRunning
     })
 
+    const config = computed(() => {
+      const { showPreview = true } = getCurrentInstance().proxy.applicationConfig
```

Comment:
I would recommend something like this:
```javascript
const defaults = useAppDefaults({
  applicationName: 'markdown-editor'
})
const { applicationConfig } = defaults

const config = computed(() => {
      const { showPreview = true } = unref(applicationConfig)
})

...

return {
  ...defaults

 ...
}
```

There's no need to go through the current instance